R-package/DESCRIPTION

```diff
@@ -0,0 +1,25 @@
+Package: RGF
+Type: Package
+Title: Regularized Greedy Forest
+Version: 1.0.5
+Date: 2018-08-09
+Authors@R: c( person("Lampros", "Mouselimis", email = "mouselimislampros@gmail.com", role = c("aut", "cre")), person("Ryosuke", "Fukatani", role = "cph", comment = "Author of the python wrapper of the 'Regularized Greedy Forest' machine learning algorithm"), person("Tong", "Zhang", role = "cph", comment = "Author of the 'Regularized Greedy Forest' and of the Multi-core implementation of Regularized Greedy Forest machine learning algorithm"), person("Rie", "Johnson", role = "cph", comment = "Author of the 'Regularized Greedy Forest' machine learning algorithm") )
```

Comment:
@StrikerRUS Your name should be added? You are also "Author of the python wrapper of the 'Regularized Greedy Forest' machine learning algorithm".

Reply:
@fukatani Thank you! It'll be an honor for me.